python/ql/test/experimental/dataflow/coverage/test.py

```diff
@@ -39,93 +42,241 @@ def test_floatnumber_literal():
 
 def test_imagnumber_literal():
     x = 42j
-    SINK(x)
+    SINK(x) # Flow missing
```

Comment:
I would guess that this is correct - the comments should indicate that this missing flow is expected somehow.

Reply:
Flow is missing here because our analysis does not represent complex numbers. I believe we want to eventually?

Reply:
Aha. I presumed that `42j` was just another `NONSOURCE`.